inventory/environ.py

```diff
@@ -229,7 +229,9 @@ def getLicenses(vars_scope):
     if vars_scope["splunk"]["license_uri"] and '*' in vars_scope["splunk"]["license_uri"]:
         vars_scope["splunk"]["wildcard_license"] = True
     vars_scope["splunk"]["nfr_license"] = os.environ.get("SPLUNK_NFR_LICENSE", "/tmp/nfr_enterprise.lic")
-    vars_scope["splunk"]["ignore_license"] = bool(os.environ.get("SPLUNK_IGNORE_LICENSE"))
+    vars_scope["splunk"]["ignore_license"] = False
+    if os.environ.get("SPLUNK_IGNORE_LICENSE", "").lower() == "true":
```

Comment:
I'm fine if we don't want to support other bool representations - i.e. 0,1 - but we should probably make sure that its documented that we only support "true" and "false"

Reply:
Sure thing, I'll make another PR for docs because I think there's a lot we need to publish for this next release. On this point, there's a couple other boolean-like checks like this we do throughout, might be useful to have a common function to address the truthy-like value during this parsing.